bin/oscapd-evaluate

```diff
@@ -103,6 +103,9 @@ def cli_scan(args, config):
     scanned_targets = []
     failed_targets = []
 
+    if not args.profile.startswith("xccdf_org.ssgproject.content_profile_"):
+        args.profile = "xccdf_org.ssgproject.content_profile_" + args.profile
```

Comment:
NACK. This will break with changes like https://github.com/OpenSCAP/scap-security-guide/pull/1771

We definitely need to do better. Let's not allow short profile IDs in daemon for now. We have a better fix for it coming to openscap sometime soon.